src/content/config.ts

```diff
@@ -26,9 +26,21 @@ const events = defineCollection({
                     end: z.date().optional(),
                 })
                 .optional(),
-            locationName: z.string().optional(),
-            locationURL: z.string().url().or(z.string().startsWith('#')).or(z.array(z.string().url())).optional(),
-            locationLatLng: z.array(z.number(), z.number()).optional(),
+            locations: z
+                .array(
+                    z.object({
+                        locationName: z.string().optional(),
+                        locationURL: z
```

Comment:
What do folks think about renaming these to just `name`, `url` etc now that they're nested? The `location` seems a bit duplicative and unnecessary..